optimum/habana/transformers/training_args.py

```diff
@@ -190,6 +190,11 @@ class GaudiTrainingArguments(TrainingArguments):
         },
     )
 
+    context_parallel_size: Optional[int] = field(
+        default=1,
+        metadata={"help": ("Determines how many ranks are divided into context parallel group.")},
+    )
```

Comment:
Should we mention this new arg in the documentation to let people know how to use it? Or is the support rather experimental so far?

Reply:
We can add this to documentation.
However, I am not sure about the issues related to DistributedAttention esp. for models other than Llama.

Reply:
Okay, let's keep it that way for now then :+1: